tests/phpunit/unit/InputToEntity/FullWikibaseItemInputTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace MediaWiki\Extension\WikibaseReconcileEdit\Tests\Unit\InputToEntity;
+
+use Deserializers\Deserializer;
+use MediaWiki\Extension\WikibaseReconcileEdit\InputToEntity\FullWikibaseItemInput;
+use MediaWiki\Extension\WikibaseReconcileEdit\MediaWiki\ReconciliationException;
+
+/**
+ * @covers \MediaWiki\Extension\WikibaseReconcileEdit\InputToEntity\FullWikibaseItemInput
+ */
+class FullWikibaseItemInputTest extends \MediaWikiUnitTestCase {
+
+	public function provideTestGetItem() {
+		yield 'Empty' => [
+			[]
+		];
+		yield 'Type Property' => [
+			[
+				'wikibasereconcileedit-version' => '0.0.1/full',
```

Comment:
The version isn’t used by the `FullWikibaseItemInput`, right? I’d leave it out to avoid confusion (it’s used by a different class, `EditRequestParser`, which we don’t test here).

Reply:
Yes, that's true.